src/style.scss

```diff
@@ -18,6 +18,9 @@
 
 $auro-inset-spacing-options: lg;
 $auro-inset-directions:'--squish';
+$auro-rgb-color-base-black-03: rgba(0, 0, 0, .03);
+$auro-rgb-color-base-white-03: rgba(255, 255, 255, .03);
```

Comment:
I am confused as to why we need this? Are there not tokens for this? We really shouldn't be inventing colors in the scope of the element. 

If this PR is also using #159 why is this work not in the other PR?